dushimeemma/kigc-esas-frontend
# #177838151 create room

### What does this PR do
- Allow a dep officer to create room
### Work to be done
- Create room feature
- Display rooms created thus far
### How to be manually tested
- Clone this repo and switch your working directory
- Install all dependencies and start the server
- In your browser login, then on the dashboard hit on rooms
> Your should see created room thus far
### Relevant pivotal tracker id
[#177838151](https://www.pivotaltracker.com/story/show/177838151)
### Screenshots
![image](https://user-images.githubusercontent.com/44426710/115681555-efc24a00-a354-11eb-8998-2e30f5154ea3.png)
